specification/sections/01-flag-evaluation.md

```diff
@@ -119,32 +135,31 @@ MyStruct myStruct = client.getObjectValue<MyStruct>('structured-flag', { text: '
 
 See [evaluation context](./03-evaluation-context.md) for details.
 
-##### Condition 1.3.2
+#### Condition 1.3.2
 
 > The implementation language differentiates between floating-point numbers and integers.
 
-###### Conditional Requirement 1.3.2.1
+##### Conditional Requirement 1.3.2.1
 
 > The client **SHOULD** provide functions for floating-point numbers and integers, consistent with language idioms.
 
-```go
-// example in GO
-GetIntValue(flag string, defaultValue int64, evalCtx EvaluationContext, options ...EvaluationOption) (int64, error)
+```java
+int getIntValue(String flag, int defaultValue);
 
-GetFloatValue(flag string, defaultValue float64, evalCtx EvaluationContext, options ...EvaluationOption) (float64, error)
+long getFloatValue(String flag, long defaultValue);
```

Comment:
I move we try to use Java examples as they are generally well-understood and familiar to most.